apps_product_creator/models/product_product.py

```diff
@@ -260,33 +257,16 @@ def _get_version_attribute(self, values):
         )
         return attribute
 
-    @api.multi
     @api.depends(
-        "image_variant",
-        "product_tmpl_id.image",
+        "image_variant_1920",
+        "product_tmpl_id.image_1920",
         "image_module",
         "odoo_module_version_id",
     )
     def _compute_images(self):
```

Comment:
You haven't checked this as commented: this method doesn't exist anymore in v13, so it's dead code.